src/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+  margin: inherit;
+}
```

Comment:
It is better to write directly `margin: 0;`

Reply:
okay)